imap_processing/ultra/l1b/de.py

```diff
@@ -163,16 +158,13 @@ def calculate_de(
         ancillary_files,
     )
     start_type[valid_indices] = de_dataset["start_type"].data[valid_indices]
-
-    (
-        event_times[valid_indices],
-        spin_starts[valid_indices],
-        _,
-    ) = get_eventtimes(
-        de_dict["spin"][valid_indices],
-        de_dataset["phase_angle"].data[valid_indices],
+    (event_times, spin_starts, spin_number) = get_eventtimes(
```

Comment:
Can you put an underscore separating event and times in all of these functions?
```suggestion
    (event_times, spin_starts, spin_number) = get_event_times(
```